service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/OthersControllerAboutToSubmitTest.java

```diff
@@ -48,6 +48,23 @@ void shouldOnlyRemoveConfidentialDetailsWhenOtherIsMarkedAsConfidential() {
         assertThat(unwrapElements(caseData.getConfidentialOthers())).containsOnly(confidentialOther());
     }
 
+    @Test
+    void shouldNotSaveFirstOtherWhenItIsEmpty() {
```

Comment:
maybe this test could be a little clearer, since it sounds like the save should fail rather than the first Other being overwritten by any following non-empty other Others

Reply:
is it worth including a 2nd `additionalOthers()` here to show that the values 'move up one', and it's not that the `getFirstOther()` and `getAdditionalOthers()` just flip values? so like
```
        assertThat(caseData.getOthers().getFirstOther()).isEqualTo(additionalOthers.get(0).getValue());
        assertThat(caseData.getOthers().getAdditionalOthers()).isEqualTo(additionalOthers.get(1).getValue());
```

Reply:
Updated the test to add another element and assert the size and elements of the collection